spec/ecommerce/resources/account_order_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+describe Ecommerce::Resources::AccountOrder do
+  describe '.find_all', vcr: true do
+    context 'when success' do
+      subject { described_class.find_all('e5732007-7989-4372-8e72-9ec8cf6ee046') }
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.